src/reducers/selectionReducer.js

```diff
@@ -35,6 +35,10 @@ const initialState = {
 };
 
 export default (state=initialState, { id: cardId, type }) => {
+  if(!cardId) {
```

Comment:
Doing it based on `id` seems like it might be the wrong approach. All sorts of actions will have an `id`. We should either filter which actions trigger this, or handle the case when it is undefined.